solr/core/src/java/org/apache/solr/cloud/ReplicateFromLeader.java

```diff
@@ -133,18 +126,25 @@ public static String getCommitVersion(SolrCore solrCore) {
   }
 
   /**
-   * Determine the poll interval for replicas based on the auto soft/hard commit schedule
+   * Determine the poll interval for replicas based on the auto soft/hard commit schedule or
+   * configured commit poll interval
    *
    * @param uinfo the update handler info containing soft/hard commit configuration
    * @return a poll interval string representing a cadence of polling frequency in the form of
-   *     hh:mm:ss
+   *     hh:mm:ss, never <code>null</code>
```

Comment:
Nice, didn't know that `never` is a special value. Perhaps we could add test coverage for it too whilst in the neighbourhood? Thanks!

Reply:
Really? Sure!

Reply:
Ah, I mis-read the comment and thought that "never" was a string value that could be returned when actually the comment was about a `null` value meaning never.

Yes, if null is never returned then no need to comment about it in the javadocs (and then also no need for the caller to consider what to do when null is returned).